src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
Your change affects how frequently (during recursion) the `captureOrPromotion` condition is satisfied, although you didn't change those lines of code.

I'm attempting to identify a more general algorithm than 19ed4888dbd888c6c381e4336cf88abb3c67322e (since conditions for deepening search for standard and giveaway chess significantly differ).

Good point about `ttCapture`, that sounds correct.